src/components/modals/ParticipantsModal.tsx

```diff
@@ -271,52 +272,53 @@ export default function ParticipantsModal({
             holders
           </Trans>
         </h4>
+        <Space direction="vertical">
```

Comment:
random change as I was passing by. Uses `Callout` and wraps the thing in `Space`